packages/svelteui-core/src/components/IconRenderer/IconRenderer.stories.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+	import { Meta, Story, Template } from '@storybook/addon-svelte-csf';
+	import { InfoCircled } from 'radix-icons-svelte';
+	import Alert from '../Alert/Alert.svelte';
+	import Button from '../Button/Button.svelte';
+	import Input from '../Input/Input.svelte';
+	import Menu from '../Menu/Menu.svelte';
+	import MenuItem from '../Menu/MenuItem/MenuItem.svelte';
+	import Notification from '../Notification/Notification.svelte';
+	import Tab from '../Tabs/Tab/Tab.svelte';
+	import Tabs from '../Tabs/Tabs.svelte';
+	import IconRenderer from './IconRenderer.svelte';
+
+	const iconSvg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+	const iconPath = document.createElementNS('http://www.w3.org/2000/svg', 'path');
+
+	iconSvg.setAttribute('fill', 'none');
+	iconSvg.setAttribute('viewBox', '0 0 24 24');
+	iconSvg.setAttribute('stroke', 'currentColor');
+	iconSvg.classList.add('post-icon');
+
+	iconPath.setAttribute(
+		'd',
+		'M13.828 10.172a4 4 0 00-5.656 0l-4 4a4 4 0 105.656 5.656l1.102-1.101m-.758-4.899a4 4 0 005.656 0l4-4a4 4 0 00-5.656-5.656l-1.1 1.1'
+	);
+	iconPath.setAttribute('stroke-linecap', 'round');
+	iconPath.setAttribute('stroke-linejoin', 'round');
+	iconPath.setAttribute('stroke-width', '2');
+
+	iconSvg.appendChild(iconPath);
+</script>
+
+<Meta
+	title="Components/IconRenderer"
+	component={IconRenderer}
+	argTypes={{
+		override: { control: false },
+		className: { control: false },
+		icon: { control: false },
+		iconSize: { control: false },
+		iconProps: { control: false }
+	}}
+/>
+
```

Comment:
If it's not asking too much, it would be cool to add a story where its only the IconRenderer (both with component and SVG) where it is possible to change the size of the icon (it will make it easier to test and try out stuff)

Reply:
I've done this by splitting the stories;
`IconRenderer.stories.svelte` is the Story for the actual component
`IconRendererUsage.stories.svelte` is the story demonstrating the component being used in other components

Reply:
Thank you, good idea 👍